data/dfndata/npc/daemons.dfn

```diff
@@ -361,4 +374,18 @@ RUNS
 RACE=10
 FLAG=NEUTRAL
 ORIGIN=uor
+}
+
+//oni - Base NPC with shared DFN tags
+[base_oni]
+{
+GET=base_daemons
+NAME=#//oni
+ID=0x0f1
+ORIGIN=se
+}
+
+[oni]
+{
+GETSE=oni_se
```

Comment:
Add one of these for each expansion, so adding NPC with wrong expansion selected doesn't break the server. We can update where these point at when we have definitions for different eras:

GETUO=oni_se
GETT2A=oni_se
GETUOR=oni_se
GETTD=oni_se
GETLBR=oni_se
GETAOS=oni_se
GETSE=oni_se
GETML=oni_se
GETSA=oni_se
GETHS=oni_se
GETTOL=oni_se